lib/shortcuts/api.md

```diff
@@ -92,8 +92,8 @@ Log Group, a Role, an Alarm on function errors, and the Lambda Function itself.
         the name of the condition here. See [AWS documentation][78]. (optional, default `undefined`)
     -   `options.DependsOn` **[String][60]** Specify a stack resource dependency
         to this Lambda function. See [AWS documentation][79]. (optional, default `undefined`)
-    -   `options.Statement` **[Array][68]&lt;[Object][59]>** an array of policy statements
-        defining the permissions that your Lambda function needs to execute. (optional, default `[]`)
+    -   `options.Statement` **[Array][68]&lt;[Object][59]>** Policy statements that will be added to a generated IAM role defining the permissions your Lambda function needs to run. (optional, default `[]`)
+    -   `options.RoleARN` **[String][60]** If specified, the Lambda function will use this role instead of creating a new role. (optional, default `undefined`)
```

Comment:
I'd suggest that these should be exclusive of each other. Specifying both should throw an error -- I think it's lead to ambiguity and confusion if we tried to support both at once.

Reply:
```suggestion
    -   `options.Statement` **[Array][68]&lt;[Object][59]>** Policy statements that will be added to a generated IAM role defining the permissions your Lambda function needs to run. (optional, default `[]`). _Do not use this option when specifying your own role via RoleARN._
    -   `options.RoleARN` **[String][60]** If specified, the Lambda function will use this role instead of creating a new role. (optional, default `undefined`). _If this option is specified, do not use the Statement option; add the permissions you need to your Role directly._
```

Something like that? Sorry for overlooking the restriction.

Reply:
committed basically this, but to the JSDoc itself